pkgs/games/fs2-knossos/default.nix

```diff
@@ -0,0 +1,133 @@
+{ lib
+, buildPythonApplication
+, fetchFromGitHub
+, mkYarnModules
+, makeDesktopItem
+, copyDesktopItems
+, makeWrapper
+, pythonOlder
+, setuptools
+, pyqt5
+, pyqtwebengine
+, six
+, requests
+, requests-toolbelt
+, raven
+, ply
+, semantic-version
+, token-bucket
+, ninja
+, SDL2
+, openal
+, p7zip
+, nodejs
+, qt5
+, appimage-run
+}:
+
+let
+  pname = "fs2-knossos";
+  version = "0.14.3";
+
+  yarnDeps = mkYarnModules {
+    pname = "${pname}-yarn-deps";
+    inherit version;
+    packageJSON = ./package.json;
+    yarnLock = ./yarn.lock;
+    yarnNix = ./yarn.nix;
+  };
```

Comment:
Can we use the newer yarn/npm tooling that does not reequire commited nix and lock files?

Reply:
I'm not aware of this new tooling, this was the only way I could find via looking through nixpkgs.